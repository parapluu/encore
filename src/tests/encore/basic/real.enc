-- This file was automatically converted by encorec

fun foo(x : real) : real
  x
end
active class Main
  def main() : unit
    val pi = 3.141593
    println("Pi is something like {}", pi)
    println("Ints can be promoted to reals: {}", foo(42))
  end
end
