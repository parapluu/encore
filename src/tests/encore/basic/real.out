Pi is something like 3.141593
Ints can be promoted to reals: 42.000000
